### PR TITLE
Specify additional environment files to add to the service-overrides.conf

### PR DESCRIPTION
**Pull request by emalihin**
It should be useful to include this commit to facilitate integration of Docker with other tools.

### CODE REVIEW

File: manifests/params.pp [hunk 1]
```diff
@@ -28,6 +28,7 @@
   $selinux_enabled                   = undef
   $socket_group                      = undef
   $labels                            = []
+  $daemon_environment_files          = []
```
Comment (garethr) [inline manifests/params.pp]: It doesn't look like this variable is used. The code is  little inconsistent at the moment with blank defaults. For the moment I'd just delete this line and leave the default in the init.pp/

**Comment by garethr** [comment]
Thanks, nice idea and thanks for explaining the usecase. I left one picky comment to delete the line in params which isn't used, but happy to merge once that's done.

**Comment by garethr** [comment]
@emalihin thanks. If you could squash the commits and rebase against master (as this now hits a merge conflict unfortunately) I'll get this merged and into the next release. Cheers

**Comment by emalihin** [comment]
Thanks @garethr this should be ready now.

**Comment by garethr** [comment]
@emalihin thanks, good job.

**Pull request merged**